training/nvidia-bootc/Containerfile

```diff
@@ -191,6 +191,9 @@ RUN --mount=type=secret,id=${INSTRUCTLAB_IMAGE_PULL_SECRET}/.dockerconfigjson \
     else \
          IID=$(sudo podman --root /usr/lib/containers/storage pull ${INSTRUCTLAB_IMAGE}); \
     fi
+    chmod -R a+rx /usr/lib/containers/storage
```

Comment:
This was our original approach but we abandoned it because this makes all the files even inside the container executable

Reply:
https://github.com/containers/ai-lab-recipes/pull/705

Reply:
There was [this](https://github.com/containers/ai-lab-recipes/pull/705#discussion_r1698613551) suggestion but I never tried it, IDK

Reply:
@cgwalters's suggestion to use `chmod a+rX` is nice. It will avoid adding `x` permission to config files or example scripts.